documentation/contributing/future_projects/index.markdown

```diff
@@ -15,62 +15,101 @@ This page lists potential projects that would greatly benefit the MoveIt project
 
 ## TrajOpt Integration and Related Work on Trajectory Optimization Methods
 
-TrajOpt is a motion planning framework from UC Berkeley for generating robot trajectories by local optimization. Integration of TrajOpt into the MoveIt framework would require API changes to how MoveIt stores and accesses information about the kinematic chain and changes to the MoveIt collision checking API to support integration with the Bullet collision checker for convex to convex contact checking.
+- List of prerequisites: ROS, some motion planning background
```

Comment:
"some motion planning background" sounds a bit unspecific

Reply:
I don't want to scare students away. Obviously, all things being equal I prefer student with a lot of experience.  Students are encouraged in their application to describe their relevant experience. See my instructions for students in the GSoC application.